doc/man3/EVP_EncryptInit.pod

```diff
@@ -613,8 +613,11 @@ the tag length has not been set.
 
 Return the block size of a cipher when passed an B<EVP_CIPHER> or
 B<EVP_CIPHER_CTX> structure. The constant B<EVP_MAX_BLOCK_LENGTH> is also the
-maximum block length for all ciphers. A value of 0 is returned if the cipher
-has not been properly initialized with a call to B<EVP_CipherInit>.
+maximum block length for all ciphers.
+A value of 0 is returned if, with B<EVP_CIPHER_get_block_size()>, the cipher
+I<e> is B<NULL>, or, with B<EVP_CIPHER_CTX_get_block_size()>, the context
+I<ctx> is B<NULL> or has not been properly initialized with a call to
```

Comment:
Explicitly documenting that NULL ctx is tolerated might not be a good idea as older OpenSSL versions won't tolerate it. If anything, there should be a HISTORY section entry.

Reply:
Well, it wasn't promised in the docs for the earlier releases, but I'm willing to clutter the HISTORY with trivia. :-)

Reply:
I would document the current API contract and you could not that it wasn't previously supported in History - however I do not think that the documentation of the *current* release is meant to document the behaviour of all previous releases and all the changes over time - that becomes unworkable. 

If you want to know the docs for a release, then you read those docs.